### ISSUE TITLE
Lots of whitespace around SVGs

### ISSUE THREAD

**Issue by jorisvervuurt** [basmilius/weather-icons#20]
First of all, thanks for creating Meteocons!

The (animated) SVGs have a lot of (transparent) whitespace around them. Is there a reason for that? I'd much prefer SVGs without any extra space around them, as it makes it so much easier to work with them.

**Comment by jorisvervuurt** [comment]
@basmilius any chance for a reply on this issue? 😄 Thanks!

**Comment by basmilius** [comment]
Hi!

From designing and animating the icons it's easier for me to have a 512x512 pixel canvas. I do however agree that the whitespace is a lot sometimes though.

I've been looking into node packages that can crop svg's, but haven't found anything that I can use. Another concern is that animations could get cut off when the svg's are cropped.

If you have any ideas for a solution, please let me know 🙏🏼

**Comment by jorisvervuurt** [comment]
Unfortunately, I don't really have a solution either. I'm currently manually cropping the static SVGs as cropping the animated ones doesn't work (I'm using Affinity Designer).

Is it hard to crop them just before exporting? :) I'd live to be able to use the animated SVGs (also tried the Lottie versions), but it's not an option due to the whitespace.![image](https://user-images.githubusercontent.com/13164041/188275729-0a9e0b50-ec42-4aba-a4ec-7af105ac12e0.jpeg)

**Comment by basmilius** [comment]
Yes, it's hard to crop them before exporting because of some animations that are using elements that go out of their view box.

Cropping static versions of the icons could be an option, but I'd like them to be the same. I could look into providing whitespace information like so:

```json
{
  "[icon-name]": {
    "top": 10,
    "left": 10,
    "right": 10,
    "bottom": 10
  }
}
```
